src/client/client.tsx

```diff
@@ -20,6 +20,7 @@ export function fetchAllEvents(): Promise<Event[]> {
 }
 
 export function fetchMember(id: Number): Promise<Member> {
+  console.log(id);
```

Comment:
```suggestion
```

Reply:
Haven't fully tested everything bc there are updates from main that haven't been pulled yet. Will re-review after the branch is updated!
User preferences load correctly based on the corresponding example user.